### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aicarmic**
Updates to readme, aiming to help new users and those running into common issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,17 @@ The HashiCorp Terraform Visual Studio Code (VS Code) extension adds syntax highl
 - Closes braces and quotes
 - Includes `for_each` and `variable` syntax shortcuts (`fore`, `vare`, `varm`)
 
+## Getting Started
+
+**IMPORTANT:** After installing, you must perform a `terraform init` to provide `terraform-ls` with an up-to-date Provider schema. The language-server will not work correctly without first completing this step!
```
Comment (radeksimko) [inline README.md:17]:
```suggestion
**IMPORTANT:** After installing, you must perform a `terraform init` to provide `terraform-ls` with an up-to-date provider schemas. The language server will not work correctly without first completing this step!
```
(minor nits) We never capitalize `provider` and technically there can be schemas for more than a single provider.

File: README.md [hunk 2]
```diff
@@ -12,6 +12,17 @@ The HashiCorp Terraform Visual Studio Code (VS Code) extension adds syntax highl
...
+
+1. Install the extension from the Marketplace
+1. Reload VS Code after the installation (click the reload button next to the extension).
```
Comment (radeksimko) [inline README.md:20]: Maybe too much context - unsure - but we could link to https://github.com/hashicorp/terraform-ls/issues/222 here - i.e. this step should not be necessary anymore once that issue is resolved

File: README.md [hunk 3]
```diff
@@ -12,6 +12,17 @@ The HashiCorp Terraform Visual Studio Code (VS Code) extension adds syntax highl
...
+1. Perform a `terraform init` to provide `terraform-ls` with an up-to-date Provider schema
```
Comment (radeksimko) [inline README.md:21]:
```suggestion
1. Perform a `terraform init` to provide `terraform-ls` with an up-to-date provider schemas
```
AFAIK we never capitalize `provider` anywhere.

File: README.md [hunk 4]
```diff
@@ -12,6 +12,17 @@ The HashiCorp Terraform Visual Studio Code (VS Code) extension adds syntax highl
...
+1. Open your desired Workspace and/or the root folder containing your Terraform files. Note: see [*Known Issues*](#known-issues) below about multi-folder workspaces
```
Comment (radeksimko) [inline README.md:22]:
```suggestion
1. Open your desired workspace and/or the root folder containing your Terraform files. Note: see [*Known Issues*](#known-issues) below about multi-folder workspaces
```

I may not be fully familiar with the VS Code naming convention here, but is "workspace" the official name for it? Either way I think it probably shouldn't be capitalized.

File: README.md [hunk 5]
```diff
@@ -12,6 +12,17 @@ The HashiCorp Terraform Visual Studio Code (VS Code) extension adds syntax highl
...
+1. Depending on your settings in VS Code, completion will start automatically (if not inside quotes/string literal, on certain trigger characters), or you can explicitly trigger completion via keyboard combination (Ctrl+Space on Windows, control+space on Mac).
```
Comment (radeksimko) [inline README.md:23]: Similar to above we could link to https://github.com/hashicorp/vscode-terraform/issues/373 here, but I'm unsure if the context is helpful.

Comment (radeksimko) [inline README.md:23]:
```suggestion
1. Depending on your settings in VS Code, completion will start automatically (if not inside quotes/string literal, on certain trigger characters), or you can explicitly trigger completion via keyboard combination (`Ctrl+Space` by default).
```

**Review by radeksimko — COMMENTED** [review]
I think we should also mention how to enable the formatting because AFAIK it doesn't get enabled by default which is probably what's tripping up most people, e.g. add

```
    "[terraform]": {
        "editor.formatOnSave": true
    },
```
I'm not sure if there is a GUI equivalent to this.

File: README.md (hunk 2 above)
Comment (aicarmic) [reply README.md:20]: I'd say we can leave out for these instructions, and just remove it from the steps once we enhance/add monitoring

**Comment by aicarmic** [comment]
@radeksimko merged in most of your suggestions, thanks for taking a look

**Review by radeksimko — APPROVED** [review]

**Pull request merged**